File: 03_Exception/WriteText.java
import java.io.*;

public class WriteText {
    public static void main(String[] args) {
        try {
            FileWriter out = new FileWriter("output.txt");
            String lf = System.lineSeparator();
            String text = "Hello, World." + lf + "  from WriteText.java";
            out.write(text);
            out.close();
        } catch (IOException e) {
            System.out.println("An error occurred.");
        }
    }
}

File: 03_Exception/WriteText.c
#include <stdio.h>

int main(void) {
    FILE *fp;
    char *text = "Hello, world.\n  from WriteText.c";
    int i;

    fp = fopen("output.txt", "w");
    if (fp == NULL) {
        printf("ファイルを開けません。\n");
        return 1;
    }

    fputs(text, fp);
    fprintf(fp, "%s\n", text);

    fclose(fp);

    return 0;
}

